modules/dxp/apps/app-builder-workflow/app-builder-workflow-web/src/main/resources/META-INF/resources/js/pages/apps/edit/sidebar/DataAndViewsTab.es.js

```diff
@@ -76,7 +90,13 @@ const SelectFormView = (props) => {
 		},
 	};
 
-	return <SelectDropdown {...props} />;
+	return (
+		<div className="d-flex">
+			<SelectDropdown {...props} />
+
+			<OpenButton {...props.openProps} />
```

Comment:
Hey @naraandrade, `openButtonProps` sounds better than `openProps`, what do you think @gabrielpramos ?

Reply:
@naraandrade I think you can prevent passing of `openButtonProps` to `selectDropdown`, just deconstructing `{openButtonProps, ...props}` on the component API, what do you think?